contracts/StarfleetBounty.sol

```diff
@@ -0,0 +1,84 @@
+pragma solidity 0.6.10;
+
+import "./IERC677.sol";
+import "openzeppelin-solidity/contracts/math/SafeMath.sol";
+import "openzeppelin-solidity/contracts/access/Ownable.sol";
+
+contract StarfleetBounty is Ownable {
+
+    using SafeMath for uint256;
+    IERC677 token;
+
+    // TODO Determine fixed value for token contract
+    address public constant TRAC_TOKEN_ADDRESS = 0x0000000000000000000000000000000000000000;
+
+
+    // Official start time of the bounty period
+    uint256 public constant BOUNTY_PERIOD_LENGTH = 90 days;
+    uint256 public bounty_period_end;
+    bool public withdrawals_enabled = false;
```

Comment:
This variable is not used in the withdrawTokens function - we should add require(withdrawals_enabled)